src/App.stories.tsx

```diff
@@ -128,6 +128,60 @@ LoggedInHiddenNoPicks.story = {
 	name: 'when logged in, with no picks and not expanded',
 };
 
+export const LoggedIn = () => (
+	<div
+		css={css`
+			width: 100%;
+			max-width: 620px;
+		`}
+	>
+		<App
+			shortUrl="p/abc123"
+			pillar={ArticlePillar.News}
+			isClosedForComments={false}
+			user={aUser}
+			baseUrl="https://discussion.theguardian.com/discussion-api"
+			additionalHeaders={{
+				'D2-X-UID': 'testD2Header',
+				'GU-Client': 'testClientHeader',
+			}}
+			expanded={true}
+			onPermalinkClick={() => {}}
+			apiKey=""
+		/>
+	</div>
+);
+LoggedIn.story = {
+	name: 'when logged in and expanded',
+};
+
+export const LoggedInShortDiscussion = () => (
+	<div
+		css={css`
+			width: 100%;
+			max-width: 620px;
+		`}
+	>
+		<App
+			shortUrl="p/39f5a" // Two comments"
+			pillar={ArticlePillar.News}
+			isClosedForComments={false}
+			user={aUser}
+			baseUrl="https://discussion.theguardian.com/discussion-api"
+			additionalHeaders={{
+				'D2-X-UID': 'testD2Header',
+				'GU-Client': 'testClientHeader',
+			}}
+			expanded={true}
+			onPermalinkClick={() => {}}
+			apiKey=""
+		/>
+	</div>
+);
+LoggedInShortDiscussion.story = {
+	name: 'when logged in but only wo comments made',
+};
```

Comment:
Is the reason to move it to a story that if someone changes the logic, it will show up in the visual diff, and someone would approve the change as intentional?

Reply:
No, it was because the test wasn't rendering any comments 😐 so I wasn't able to verify the logic I was adding. I figured if the storybook tests are working it makes sense to favour using them rather than trying to support two types of test doing the same thing